Sources/StreamVideo/WebRTC/v2/VideoCapturing/ActionHandlers/Camera/CameraStopCaptureHandler.swift

```diff
@@ -0,0 +1,38 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import AVFoundation
+import CoreMedia
+import Foundation
+import StreamWebRTC
+
+final class CameraStopCaptureHandler: StreamVideoCapturerActionHandler, @unchecked Sendable {
+
+    // MARK: - StreamVideoCapturerActionHandler
+
+    func handle(_ action: StreamVideoCapturer.Action) async throws {
+        switch action {
+        case let .stopCapture(videoCapturer):
```

Comment:
Not sure if we need separate handler for the stopping action? I see in other places start/stop are together.